src/api/Controller/contactUs.js

```diff
@@ -0,0 +1,17 @@
+const ContactUs = require('../../model/ContactUs');
```

Comment:
it seems more of like crud operations mechanism.
it is not functioning as controller here

Reply:
When we will move our db logic to one util class it will have only req , res handling